daemon/daemon_windows.go

```diff
@@ -91,10 +90,10 @@ func migrateIfDownlevel(driver graphdriver.Driver, root string) error {
 	return nil
 }
 
-func configureSysInit(config *Config, rootUID, rootGID int) (string, error) {
-	// TODO Windows.
-	return os.Getenv("TEMP"), nil
-}
+//func configureSysInit(config *Config, rootUID, rootGID int) (string, error) {
```

Comment:
Oh I meant to. I just commented out to confirm that it still compiled on Windows, forgot to actually remove it subsequently. Will push in a minute.